src/Content.php

```diff
@@ -332,6 +371,30 @@ public function setValue($key, $value)
             }
         }
 
+        if ($key == 'templatefields') {
+            $oldValue = $this->values[$key];
+            if ((is_string($value)) || (is_array($value))) {
+                if (is_string($value)) {
+                    try {
+                        $unserdata = Lib::smartUnserialize($value);
+                    } catch (\Exception $e) {
+                        $unserdata = false;
+                    }
+                } else {
+                    $unserdata = $value;
+                }
+
+                if ($unserdata !== false) {
+                    $templateContent = new Content($this->app, $this->getTemplateFieldsContentType(), array(), false);
+                    $value = $templateContent;
+                    $this->populateTemplateFieldsContenttype($value);
```

Comment:
@Pinpickle can you remember what this was supposed to be for/doing? 

The function itself never got committed… very obviously the :koala: snuck in and messed with things :wink: